commit 0215cacb306092511d5e108005822e8b5e38d07a
Merge: 7b95918 5db3738
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Sat Nov 19 01:32:40 2016 -0600

    Merge pull request #5247 from esenapaj/Re-follow-the-PR-#4634
    
    Re-follow the PR #4634 (Set language display charset in language.h)

diff --git a/Marlin/language_da.h b/Marlin/language_da.h
index 09aaa82..e1eb168 100644
--- a/Marlin/language_da.h
+++ b/Marlin/language_da.h
@@ -31,8 +31,6 @@
 #define LANGUAGE_DA_H
 
 #define MAPPER_C2C3
-// Define SIMULATE_ROMFONT to see what is seen on the character based display defined in Configuration.h
-//#define SIMULATE_ROMFONT
 #define DISPLAY_CHARSET_ISO10646_1
 
 #define WELCOME_MSG                         MACHINE_NAME " er klar"
